### DISCUSSION TITLE
Yanking to system Clipboard doesn't work on wayland (sway)

### DISCUSSION THREAD

**Discussion by ClarkAndrews** [helix-editor/helix#12070] [Troubleshooting]
So I downloaded wl-clipboard. /usr/bin is added in my PATH and wl-copy, wl-paste are in incliuded inside those. 
still "space + y" and then copying in another window doesn't work.

appreciate every help I can get

**Comment by rhogenson** [comment]
What does `hx --health clipboard` show?

**Reply by RocketRene** [reply]
I'm on the fedora sway spin. I installed `wl-clipboard` and it works fine.

```
$ hx --health clipboard                                                                                                                 
System clipboard provider: wl-paste+wl-copy
```

**Reply by aggsol** [reply]
@RocketRene Did you find a solution? I have the same issues.

**Reply by RocketRene** [reply]
Did you Install wl-clipboard?

**Reply by aggsol** [reply]
Yes, it's version 2.2.1-1build1
